site/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -28,19 +28,25 @@ export const useStyles = makeStyles((theme) => ({
   },
 }))
 
+interface LocationState {
+  isRedirect: boolean
+}
+
 export const LoginPage: React.FC = () => {
   const styles = useStyles()
   const location = useLocation()
   const xServices = useContext(XServiceContext)
   const [authState, authSend] = useActor(xServices.authXService)
   const isLoading = authState.hasTag("loading")
   const redirectTo = retrieveRedirect(location.search)
+  const locationState = location.state ? (location.state as LocationState) : null
+  const isRedirected = locationState !== null ? locationState.isRedirect : false
```

Comment:
Can you simplify like:
```suggestion
  const isRedirected = !locationState ? locationState.isRedirect : false
```
Or are we using `null` for a reason?

Reply:
Yes, I think I can just use `locationState`.